### ISSUE TITLE
Training LightCNN-29 with CASIA-WebFaces

### ISSUE THREAD

**Issue by lyatdawn** [tornadomeet/mxnet-face#45]
Use the code in verification dir to train LightCNN-29. When Training LightCNN-29 with CASIA, we can see: the training loss decreases slowly, and the training/valing accuracy increase also slowly. And, training/valing accuracy is always close to 0.0000. 
Could you give some advises to speed the training phase, and some for increase the train/val accuracy? 
For example, the initialization methods for init the weights, the learning rate scheduler, and so on. 
Thanks
